docs/reference/modules/discovery/discovery-settings.asciidoc

```diff
@@ -5,10 +5,13 @@ Discovery and cluster formation are affected by the following settings:
 
 `discovery.seed_hosts`::
 
-    Provides a list of master-eligible nodes in the cluster. Each value has the
-    format `host:port` or `host`, where `port` defaults to the setting
-    `transport.profiles.default.port`. Note that IPv6 hosts must be bracketed.
-    The default value is `["127.0.0.1", "[::1]"]`. See <<unicast.hosts>>.
+    Provides a list of the addresses of the master-eligible nodes in the
+    cluster. Each address has the format `host:port` or `host`, where `port`
+    defaults to the setting `transport.profiles.default.port` falling back to
+    `transport.port` if not set; if neither is set then the default port is
+    `9300`. Note that IPv6 addresses must be bracketed. May also be a single
+    string containing all the addresses separated by commas. The default value
```

Comment:
> May also be a single string containing all the addresses separated by commas.

It's unclear if this only applies to IPv6 addresses.

Reply:
Good point, I moved this sentence.